index.html

```diff
@@ -1,7 +1,43 @@
-
+<!DOCTYPE html>
+<hml>
+	<head>
+		<title>Introduction to the DOM</title>
+		<link rel="stylesheet" href="./styles.css">
+		<script defer="defer" src="components/Dropdown/dropdown.js"></script>
+		<script type="text/javascript" defer="defer" src="components/Tabs/tabs.js"></script>
+	</head>
+	<body>
+		<div class="section">
+		<div class="header">
+			<div class="header__dropdown header_dropdown--hover">
```

Comment:
Just out of curiousity - was there a reason you did `header_dropdown--hover` rather than `header__dropdown--hover`?? You have one underline rather than two